MultiplayerMod/Core/Logging/LogLevel.cs

```diff
@@ -1,9 +1,9 @@
 ﻿namespace MultiplayerMod.Core.Logging;
 
 public enum LogLevel {
-    Trace,
-    Debug,
-    Info,
-    Warning,
-    Error
+    Trace = 0,
```

Comment:
What's the reason for explicit values here?

Reply:
Actually didn't know that enums are comparable by default